packages/client/src/admin/SystemSetting/__tests__/SystemSetting.spec.tsx

```diff
@@ -30,7 +30,9 @@ function setup() {
               maxNode: 1000,
               maxModelDeploy: 1000,
               maxGroup: 9999,
+              maxUser: 9998,
```

Comment:
The number of *9998* is come from: `9999 - 1 (admin or myself)`?

Reply:
the number is just arbitrary number to test. TO user the different number form `maxGroup` is to make sure that i did not use the wrong variable.